packages/client/src/lobby/createReplayJSON.ts

```diff
@@ -0,0 +1,226 @@
+import * as chat from "../chat";
+import ActionType from "../game/types/ActionType";
+import ClientAction from "../game/types/ClientAction";
+import ClueType from "../game/types/ClueType";
+import { LogEntry } from "../game/types/GameState";
+import { JSONGame } from "../game/types/JSONGame";
+import ReplayState from "../game/types/ReplayState";
+import globals from "../game/ui/globals";
+
+export default function createJSONFromReplay(room: string) {
+  if (
+    globals === null ||
+    globals.store === null ||
+    !globals.state.finished ||
+    globals.state.replay === null ||
+    globals.state.replay.hypothetical === null
+  ) {
+    chat.addSelf(
+      '<span class="red">Error:</span> You can only use that command during the review of a hypothetical.',
+      room,
+    );
+    return;
+  }
+  // Anonymize the players
+  const fakePlayers = ["Alice", "Bob", "Cathy", "Donald", "Emily", "Frank"];
```

Comment:
shouldn't this be a constant somewhere, probably in @hanabi/data

Reply:
That's a good idea (btw is 'Frank' correct? I kinda guessed)

Reply:
https://github.com/Hanabi-Live/hanabi-live/commit/d443c1f717f5a282d02154bde231b3f7d0f63e80